gradle.properties

```diff
@@ -91,8 +89,8 @@ graphqlToolVersion=0.14.0-20250523-094200-b95690f
 # Protoc Tool
 protocToolVersion=1.0.0-20250528-120700-b5350a0
 
-# OpenAPI Module
-openapiToolVersion=2.3.2
+# OpenAPI Tool
+openapiToolVersion=2.4.0
```

Comment:
Do we use the same version to include `ballerina/openapi` package in the distribution?

Reply:
Added by [d0ee28f](https://github.com/ballerina-platform/ballerina-distribution/pull/6147/commits/d0ee28f70af6b982e2564f62d9d30beabac3d2a2)